internal/commands/vaultsecrets/secrets/create.go

```diff
@@ -288,6 +281,70 @@ func createRun(opts *CreateOpts) error {
 				return err
 			}
 		}
+	case Dynamic:
```

Comment:
There are constants for secret types in [secret_types.go](https://github.com/hashicorp/hcp/blob/main/internal/commands/vaultsecrets/secrets/secret_types.go). Would be good to use one set of them instead of having two?